Create CounselorBot guide in Getting Started md

ganning127 opened the pull request:
Closes #401

emsesc reviewed (changes requested):
_The README is a bit confusing in terms of finding where to get help with the bot since the dropdown here says "Get Started with CounselorBot" but doesn't talk about how to install it. The section right under then says "Not sure how to get started?" which is kind of counterintuitive._

### Suggestions
* Add a checklist (ex: step 1, install bot, step 2, how to create the repo for the course)
* Declutter the GitHub README so there aren't repetitive titles

Let me know if this is confusing!

ganning127 commented:
I removed some links in README.md and also removed the "moving into your cabin" part in the getting started file because the installing counselorbot covers that. I also created a checklist on what to do to get started

emsesc reviewed (approved):
Thanks @ganning127 ! I just made the checklist a bit more concise so it doesn't look overwhelming.
